lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -8,6 +8,15 @@ const rename = promisify(fs.rename);
 const pathToFolder = require('../../support/pathToFolder');
 const path = require('path');
 
+const FIREFOX_VERSION_JS = String.raw`return (function() {
+  var match = navigator.userAgent.match(/Firefox\/(([0-9]+\.?)*)/);
+
+  if (match)
```

Comment:
nit: let's always cuddle braces, so
```js
if (match) {
...
} else {
...
}
```
It's generally what we do across Mozilla projects, although I'm certain there's local variation.